pkg/tsdb/cloudmonitoring/cloudmonitoring.go

```diff
@@ -515,87 +478,6 @@ func interpolateFilterWildcards(value string) string {
 	return value
 }
 
-func buildFilterString(filterParts []string) string {
```

Comment:
all this code has been moved to the different interfaces rather than in this generic file